extensions/wikia/ArticleVideo/scripts/ooyala/google_ima.js

```diff
@@ -4390,7 +4414,8 @@ var globalRequire = require;
 					var eventType = google.ima.AdEvent.Type;
 					// Add listeners to the required events.
 					_IMAAdsManager.addEventListener(eventType.CLICK, _IMA_SDK_onAdClicked, false, this);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Move the declaration of "_IMA_SDK_onAdClicked" before this usage. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)